features/menu_tests.feature

```diff
@@ -0,0 +1,131 @@
+Feature: Test all primary CommCare menu options
+
+@QA
+Scenario: Test that all actions available from the home screen options menu work properly 
+
+	Then I install the ccz app at "settings_sheet_tests.ccz"
+	Then I login with username "settings.test" and password "123"
+
+	Then I open the options menu
+	Then I see the text "Update App" 
+	Then I see the text "Saved Forms"
+	Then I see the text "Change Language"
+	Then I see the text "About CommCare"
+	Then I see the text "Advanced"
+	Then I see the text "Settings"
+	Then I rotate to landscape
```

Comment:
@wpride I forgot that the reason I hadn't initially tested this in landscape mode was that it was inexplicably failing when I tried to call the I see a list that contains all of these items step after rotating, because calling that somehow made the options menu disappear (even though it otherwise definitely stays open after rotating).. Decided the easiest solution was to just switch to calling I see the text for each individual item, since that doesn't cause the same problem for whatever reason.

The potential failure point of this strategy is that on some phones in landscape mode, some of the menu items might be cut off, but calling 'I scroll until I see the text' for these items caused the same failure as above, so this seems to be the best we can do.